decidim-comments/app/models/decidim/comments/seed.rb

```diff
@@ -6,25 +6,42 @@ module Comments
     # to Seed those models in order to be able to use them in the development
     # app.
     class Seed
-      # Public: adds a random amount of comments for a given resource.
-      #
-      # resource - the resource to add the coments to.
-      #
-      # Returns nothing.
-      def self.comments_for(resource)
-        return unless resource.accepts_new_comments?
+      class << self
+        # Adds a random amount of comments for a given resource.
+        #
+        # @param resource [Object] - the Decidim resource to add the coments to.
+        #                            examples: Decidim::Proposals::CollaborativeDraft, Decidim::Proposals::Proposal,
+        #
+        # @return nil
+        def comments_for(resource)
+          return unless resource.accepts_new_comments?
 
-        Decidim::Comments::Comment.reset_column_information
+          Decidim::Comments::Comment.reset_column_information
 
-        organization = resource.organization
+          rand(0..6).times do
+            comment = create_comment(resource)
+            create_comment(comment, resource) if [true, false].sample
+          end
+        end
+
+        private
 
-        2.times do
-          author = Decidim::User.where(organization:).all.sample
+        # Creates a comment for a given resource.
+        #
+        # @private
+        #
+        # @param resource [Object] - the Decidim resource to add the coments to.
+        #                            examples: Decidim::Proposals::CollaborativeDraft, Decidim::Proposals::Proposal,
+        # @param root_commentable [Decidim::Comments::Comment|nil] - the root commentable resource. It is optional, used for making nested comments.
```

Comment:
When there is a root comment, The root commentable and commentable types are resources ( Proposals ) 
When there is a reply, the commentable is a Comment, the root commentable is a resource ( like Proposal ).

```suggestion
        # @param resource [Object] - the Decidim resource to add the comments to.
        # @param root_commentable - the root commentable resource. It is optional, used for making nested comments.
```